templates/Web/Projects/FlaskDefault/server/constants.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Just wondering why are we doing this?

Reply:
This is for Compliance - Stephane asked to have this.

Reply:
Yeoman, create-react-app, angular cli don't do this.

Reply:
Odd since CRA, Angular/Vue Cli don't say "Copyright (c) Facebook" or "Copyright (c) Google" Etc. And seems like something users would want to remove. If anything i feel that these copyright headers should be on our code, not the generated code. Although I'm not as familiar with Microsoft or Garage specific compliance standards as stephane is. I would just run the idea of having it on our code rather than the code we generate by him, if he says he does indeed want this, I will approve.

Reply:
"Look at how WindowsTS does it" -Clint Rutkas 2019

Reply:
Yeah, I confirmed with Stephane that this is for our code and not the generated code. So, I am closing this PR.